docs/user/filesystem.md

```diff
@@ -502,3 +439,70 @@ $ D_IL_REPORT=-1 LD_PRELOAD=/usr/lib64/libioil.so dd if=/dev/zero of=./bar bs=1G
     function to close stderr on exit, so for many basic commands such as cp and cat
     whilst the interception library will work it is not possible to see the summary
```

Comment:
Not in the PR, however there is probably a typo error with the word whilst

Reply:
whilst is an english word :) I confirmed with Ashley that the phase is correct.

Reply:
OK, I learn a new word :)